.aspect/bazelrc/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+"Aspect bazelrc presets; see https://docs.aspect.build/guides/bazelrc"
+
+load("@aspect_bazel_lib//lib:bazelrc_presets.bzl", "write_aspect_bazelrc_presets")
+
+write_aspect_bazelrc_presets(
+    name = "update_aspect_bazelrc_presets",
+    presets = [
+        "ci",
+        "convenience",
+        "correctness",
+        "debug",
+        "javascript",
+        "performance",
+    ],
```

Comment:
Do we want bazel6/7 presets?

Reply:
We have them. They are in `.github/workflows` as only CI deals with multiple versions of Bazel. For local development, there is a `.aspect/bazelrc/bazel7.bazelrc` symlink that is load bearing.